react/__tests__/components/BuyButton.test.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import BuyButton from '../../BuyButton'
+
+describe('<BuyButton />', () => {
+  let wrapper
+
+  const renderComponent = customProps => {
+    const props = {
+      ...customProps,
+      showToast: () => {},
+    }
+
+    const component = shallow(<BuyButton {...props}> Test </BuyButton>)
```

Comment:
We need to check out why the `render`  function is returning a strange Snapshop.

Reply:
Its was not returning strange snapshot, since the component was only an button, it was returning what was in my mock. Since I want to make a snapshot of a simple component, the more appropiate is to use `shallow`